shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -850,6 +859,149 @@ \subsection{Verifiable Random Function}
   \label{fig:rules:vrf}
 \end{figure}
 
+\clear
+
+\subsection{Decentralization Slider}
+\label{sec:decentralization-slider}
+
+The transition from the boostrap era to a fully decentralized network is explained in
+section 3.9.2 of \cite{delegation_design}.
+Key to this transition is a protocol parameter $d$ which controls how many slots are governed by
+the genesis nodes via OBFT, and which slots are open to any registered stake pool.
+The transition system introduced in this section, $\type{DSLIDE}$, covers this mechanism.
+
+The environments for this transition is:
+\begin{itemize}
+  \item The environment for the $\type{VRF}$ transition $\var{ve}$.
+  \item A mapping $\var{dsched}$ of slots to an optional genesis key.
+    In the terminology of \cite{delegation_design},
+    the slots in $\var{dsched}$ are the ``OBFT slots''.
+    A slot in this map with a value of $\Nothing$ is a non-active slots,
+    otherwise it is an active slot and its value designates the genesis key
+    responsible for producing the block.
+\end{itemize}
+
+\begin{figure}
+  \emph{DSlider environments}
+  \begin{equation*}
+    \DSliderEnv =
+    \left(
+      \begin{array}{r@{~\in~}lr}
+        \var{ve} & \VRFEnv & \text{VRF environment} \\
+        \var{dsched} & \Slot\mapsto\VKeyGen^? & \text{OBFT schedule} \\
+      \end{array}
+    \right)
+  \end{equation*}
+  %
+  \emph{DSlider Transitions}
+  \begin{equation*}
+    \_ \vdash \var{\_} \trans{dslider}{\_} \var{\_} \subseteq
+    \powerset (\DSliderEnv \times \BHeaderState \times \BHeader \times \BHeaderState)
+  \end{equation*}
+  \caption{DSlider transition-system types}
+  \label{fig:ts-types:decent-slider}
+\end{figure}
+
+\begin{figure}[ht]
+  \begin{equation}\label{eq:active-pbft}
```

Comment:
Do we need a rule for inactive OBFT, too? `gkey` is an optional key, but `BHEAD` doesn't seem to treat the `Nothing` case.

Reply:
I don't think we do. No one, neither genesis key holders nor registered stake pools, are allowed to make blocks during the inactive slots.  As it is now, the `CHAIN` rule will fail to transition on inactive slots since `DSLIDE` will not transition.